Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (1.0.0)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
```

Comment:
### [mime-types-data](https://github.com/mime-types/mime-types-data/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.2023.0218.1 → 3.2024.0206

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/mime-types-data/3.2023.0218.1/3.2024.0206)_]





<details>
<summary>Commits</summary>

A change of **23** commits. See the full changes on [the compare page](https://github.com/mime-types/mime-types-data/compare/v3.2023.0218.1...v3.2024.0206).

These are the first 10 commits:
- (7109b07) [Update JavaScript types per RFC9239 (#56)](https://github.com/mime-types/mime-types-data/commit/7109b079da2c87924066c5c3e7eb9b65ffcec9e8)
- (3d6fb33) [Start automating data updates (#57)](https://github.com/mime-types/mime-types-data/commit/3d6fb330fefe7a4697974761be286ccf4792a900)
- (e29ec74) [Fix designations because '@' is YAML-protected](https://github.com/mime-types/mime-types-data/commit/e29ec74bf7df3d89e71fc6ec6a80e81e12cb6e0b)
- (deec91c) [Fix cron specification](https://github.com/mime-types/mime-types-data/commit/deec91c45535ac11cece558c69a0a1ff170da704)
- (415e9e5) [Fix reference for peter-evans/create-pull-request](https://github.com/mime-types/mime-types-data/commit/415e9e55f0f880c1e588ad3d20771e43039dc90f)
- (f1109a5) [Stop octal behaviour](https://github.com/mime-types/mime-types-data/commit/f1109a5c76d3833bf86e49ef98dfcea90954af67)
- (520fa19) [Debug workflow](https://github.com/mime-types/mime-types-data/commit/520fa19deb9c72e5f3a6f310314652f0f8767155)
- (05f82ae) [Fix output checking](https://github.com/mime-types/mime-types-data/commit/05f82ae104e2152ec4a08f18304a2b0338eecf5d)
- (fb55f6d) [Fix PR contents](https://github.com/mime-types/mime-types-data/commit/fb55f6d5bda5f8cc3056d3313ab1b1a678fb817e)
- (0278657) [Update mime-types-data 3.2023.0808 / 2023-08-08 (#60)](https://github.com/mime-types/mime-types-data/commit/02786577194203a143e421354ee74a4a53e568cb)


</details>